quantum/dip_switch.c

```diff
@@ -70,16 +84,22 @@ __attribute__((weak)) bool dip_switch_update_mask_kb(uint32_t state) {
 
 void dip_switch_init(void) {
 #ifdef DIP_SWITCH_PINS
-#    if defined(SPLIT_KEYBOARD) && defined(DIP_SWITCH_PINS_RIGHT)
+#   ifdef SPLIT_KEYBOARD
     if (!isLeftHand) {
-        const pin_t dip_switch_pad_right[] = DIP_SWITCH_PINS_RIGHT;
         for (uint8_t i = 0; i < NUMBER_OF_DIP_SWITCHES; i++) {
             dip_switch_pad[i] = dip_switch_pad_right[i];
```

Comment:
Looks like this needs a `#ifdef DIP_SWITCH_PINS_RIGHT` block around, because `dip_switch_pad_right` is defined only conditionally.

Reply:
Added that, good point.